.github/workflows/lint.yaml

```diff
@@ -43,13 +43,43 @@ jobs:
         run: |
           git remote add upstream https://github.com/coreruleset/coreruleset
           git fetch --tags upstream
-
+          git fetch upstream main
+          echo ${{ github.base_ref }}
+          if [ -n "${{ github.base_ref }}" ]; then
+            git fetch upstream "${{ github.base_ref }}"
```

Comment:
Does this necessary? Later (in line 56) you check the same condition again.

Reply:
Yes, those are two different scripts, they know nothing of each other.